opm/input/eclipse/Units/UnitSystem.cpp

```diff
@@ -1560,8 +1566,9 @@ namespace {
         Dimension dividend = this->parseFactor( parts[0] );
         Dimension divisor = this->parseFactor( parts[1] );
 
-        if (dividend.getSIOffset() != 0.0 || divisor.getSIOffset() != 0.0)
-            throw std::invalid_argument("Composite dimensions cannot currently require a conversion offset");
+        if (!dividend.isCompositable() || !divisor.isCompositable()) {
```

Comment:
I just understood why this is necessary: If any factor of either dividend or divisor is not compositable, that would usually trigger an exception from parseFactor(), but the exception is if either dividend or divisor is a single incompositable factor, then it must be handled here.

Can you confirm this is a correct understanding?

Reply:
`dividend` and `divisor` themselves can be possibly a single atomic unit with conversion offset. In this PR, this unit can be `Temperature`. 

Function paserFactor can parse `Temperature` while not `Temperature*Pressure` (throw).  

And we do not allow either `dividend` or `divisor` contain conversion offset (like `Temperature`. ) before we think  it is possible.

Reply:
> Can you confirm this is a correct understanding?

Your understanding is correct.

Reply:
Thanks, merging.